packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -288,6 +290,22 @@ class IconButton extends StatelessWidget {
   ///  * [Feedback] for providing platform-specific feedback to certain actions.
   final bool enableFeedback;
 
+  /// Box constraints for the button.
+  ///
+  /// When unspecified, defaults to:
+  /// ```dart
+  /// const BoxConstraints(
+  ///   minWidth: kMinInteractiveDimension,
+  ///   minHeight: kMinInteractiveDimension,
+  /// )
+  /// ```
+  /// where [kMinInteractiveDimension] is 48.0, and then with visual density
```

Comment:
The default constraints ensure that the the button is accessible. Creating a button that's smaller than the minimum size is not recommended.

Reply:
Edited, with some slightly different wording.